scanners/android/utility.go

```diff
@@ -176,8 +178,13 @@ func (scanner *Scanner) generateConfigBuilder(repoAccess models.RepoAccess) mode
 		envmanModels.EnvironmentItemModel{GradlewPathInputKey: gradlewPath},
 	))
 
+	basePath := filepath.Join(projectLocationEnv, moduleEnv)
+	path := filepath.Join(basePath, "build.gradle")
+	if useKotlinBuildScript {
+		path = filepath.Join(basePath, "build.gradle.kts")
```

Comment:
nit: `android.go` already defines a constant for this filename

Reply:
I created that constant and forgot about it. 😄